pyemma/datasets/double_well_thermo.py

```diff
@@ -39,6 +39,12 @@ def __init__(self):
         self.pi[:] = self.pi / self.pi.sum()
         self.f = -_np.log(self.pi)
 
+        if random_seed is not None:
+            self._rng = _np.random.RandomState(random_seed)
```

Comment:
does this overwrite the state for all numpy users? If that is the case, is there a way to circumvent reseeding others random state?

Reply:
No, I don't think so. I tested:

```
outer_rng = np.random.RandomState(100)
outer_rng.randn(1)
dw_sampler = DoubleWellSampler(random_seed=1)
outer_rng.randn(1)
p1 = DoubleWellSampler(random_seed=1).sample(length=10)['trajs'][0]
print(np.random.randn(1))
print(outer_rng.randn(1))
```

And the `outer_rng` is unaffected by the `random_seed` argument to `DoubleWellSampler` nor its execution, and the `np.random` call is different on different runs with the same `outer_rng` and `random_seed`.

Reply:
As far as I know the random state is only for calls that come from `_rng` in place of `_np.random`.

Reply:
One consideration though is that if the same `DoubleWellSampler` object were used to run `sample` more than once without a new instance of the `Sampler`, then each result would be different.

Reply:
Thanks for testing this. Why would this be an issue in the latter case? I would expect it to be different.

Reply:
Yeah, same. Not an issue.